js/templates/purchase_template/purchaseMain.js

```diff
@@ -177,9 +176,13 @@ else{
         document.getElementById("creditValue").innerHTML="₹ "+userArray[playerChance].getCredit();
         document.getElementById("creditLimitValue").innerHTML="₹ "+userArray[playerChance].getCreditLimit();
 
-        if(!offlinePurchaseClicked)
-        ubsApp.callServerNextMove();
+        if(!offlinePurchaseClicked) {
+        // ubsApp.callServerNextMove();
```

Comment:
Can you please remove comment and logging?

Reply:
done